core/src/main/java/nl/nn/adapterframework/http/rest/MediaTypes.java

```diff
@@ -15,21 +15,43 @@
 */
 package nl.nn.adapterframework.http.rest;
 
+import java.nio.charset.Charset;
+
 public enum MediaTypes {
 
-	ANY("*/*"),
+	ANY("*/*", null),
 	TEXT("text/plain"),
 	XML("application/xml"),
 	JSON("application/json"),
-	PDF("application/pdf"),
+	PDF("application/pdf", null), //raw binary formats do not have a charset
+	OCTET("application/octet-stream", null), //raw binary formats do not have a charset
 	MULTIPART_RELATED("multipart/related"),
 	MULTIPART_FORMDATA("multipart/form-data"),
 	MULTIPART("multipart/*");
 
 	private final String mediaType;
+	private final Charset charset;
```

Comment:
Kan je hier niet net zo goed een boolean hebben met 'hasCharset'?

Reply:
Ja ik dacht ik gebruik een Charset want dan kunnen we ooit misschien nog besluiten ergens een andere default charset op te geven